README.md

```diff
@@ -24,21 +24,32 @@ Formalizing and benchmarking open problems in single-cell genomics.
 
 ## The team
 
-**Core**:
-* Scott Gigante (@scottgigante)
-* Daniel Burkhardt (@dburkhardt)
-* Malte Luecken (@LuckyMD)
-* Angela Pisco (@aopisco)
-* Olga Botvinnik (@olgabot)
-
-**Task authors** (_alphabetically_):
-* Emma Dann (@emdann) - Differential abundance 
-* Wes Lewis (@weslewis) - Differential abundance and data denoising
-* Mohammad Lotfallahi (@M0hammadL) - Label projection task
-* Qian Qin (@qinqian) - Predicting gene expression from chromatin accessibility
-* Daniel Strobl (@danielStrobl) - Batch integration
-* Michael Vinyard (@mvinyard) - Stress preservation in dimensionality reduction
-* Florian Wagner (@flo-compbio) - Data denoising
+**Core** (_alphabetically_):
+
+* Daniel Burkhardt (@dburkhardt), Cellarity
+* Robrecht Cannoodt (@rcannoodt), Data Intuitive
+* Scott Gigante (@scottgigante-immunai), ImmunAI
+* Malte Luecken (@LuckyMD), Helmholtz Munich
+* Angela Pisco (@aopisco), CZ Biohub
+
+**Task leaders** (_alphabetically_):
+
+Full tasks (>1 datasets, >5 methods):
+* Batch integration - Daniel Strobl (@danielStrobl)
+* Dimensionality reduction - Luke Zappia (@lazappi) and Michael Vinyard (mvinyard)
```

Comment:
Need the @ on mvinyard. Also are we putting co-leaders alphabetically?

Reply:
thx, yes. And yes, that was the plan.